### PR TITLE
we cooked it

### PR DESCRIPTION
**Pull request by Sugargod1**
ready for checky checky out

### CODE REVIEW

File: t04_refactored.py [hunk 1]
```diff
@@ -164,10 +164,38 @@ def team_8_adv():
...
+            isDead = True
+        else:
+            print(
+                """Playful stubborn little boy, I was once like you""")
+    else:
+        # neutral choice
+        print(
+            """Now you are presented with two choices. 
+            "One direction could lead to your death and the other solution is unknown""")
     pass
```
Comment (sheggen) [inline t04_refactored.py:198]: Remove helper text

**Comment by Sugargod1** [comment]
deleted it

File: t04_refactored.py [hunk 2]
```diff
@@ -164,10 +164,37 @@ def team_8_adv():
...
+            print("Due to your declination, you will now be forced to leave the labyrinth.")
+            isDead = True
+        else:
+            print(
+                """Playful stubborn little boy, I was once like you""")
+    else:
+        # neutral choice
+        print(
+            """Now you are presented with two choices. 
+            "One direction could lead to your death and the other solution is unknown""")
```
Comment (sheggen) [inline t04_refactored.py:197]: I gave you a function to kill the user when they die. Make sure to call it here:

`kill_if_dead(isDead)`

Comment (Sugargod1) [reply t04_refactored.py:198]: we called it in our nested if statement

**Pull request closed without merging**